src/installation/guides/chroot.md

```diff
@@ -0,0 +1,313 @@
+# Installation via chroot (x86 or x86_64)
+
+This guide details the process of manually installing Void Linux (the chroot
+method) on an x86 or x86_64 PC architecture. It is assumed that you have a
+familiarity with Linux but not necessarily that you have performed a chroot
+installation before. If you are unfamiliar with the process, this guide can be
+followed literally to create a "typical" setup (one partition is used on one
```

Comment:
"... via a chroot. This guide can be used to create a 'typical' setup, using a single partition on a single SATA/IDE/USB disk."

Reply:
yes